src/js/entry.js

```diff
@@ -183,3 +185,17 @@ function loadChrome(user) {
         }
     );
 }
+
+function checkBetaPreference() {
+    if (cookie.get('betaDefault')) {
+        const splitted = window.location.href.split('/') ;
```

Comment:
Instead of this, stay consistent with how we do it in other places: 
`window.location.pathname.split('/')[1] === 'beta'`

Reply:
I'm having trouble recreating the issue that led me to switch from that to the current method right now. But essentially it seems to drop the final part of the url in some situations like 
a link like: cloud.redhat.com/inventory/accessDenied.ext?app=inventory would redirect to cloud.redhat.com/beta/inventory/ using the path method.